vetiver/handlers/base.py

```diff
@@ -1,4 +1,3 @@
-from vetiver.handlers import base
```

Comment:
I think this (or immediately after this PR) is the good time to add a module `docstring` for all the _whats_ and the _whys_ about `Handlers`. I will do a PR after this one.

Reply:
Awesome! That would be a great addition for more context on Handlers.